### ISSUE TITLE
How to add Latex to luncher

### ISSUE THREAD

**Issue by alirezamecheng** [jupyterlab/jupyterlab-latex#105]
Hello, 
I've installed the extension and it works perfectly (Thanks for your beautiful work). 
My question is How can I add Latex to the launcher in jupyterLab?
So that jupyterlab users inform that there is such an option available to use?

**Comment by ian-r-rose** [comment]
Thanks for the kind words!

What, specfically, do you want to add to the laucher? Do you want to create a new `.tex` document? Or to add a button to launch the PDF? Or something else?

**Comment by alirezamecheng** [comment]
a new `.tex` or a button like `Diagram` or `Text File` in the following Image will be perfect.
![image](https://user-images.githubusercontent.com/29999507/51415251-742c2e00-1b8a-11e9-86eb-2c946df677d1.png)

**Comment by ian-r-rose** [comment]
Yep, I think this is a reasonable thing to add. It could also be paired with #89.
